google-cloud-clients/google-cloud-tasks/src/test/java/com/google/cloud/tasks/v2beta3/CloudTasksSmokeTest.java

```diff
@@ -0,0 +1,112 @@
+/*
+ * Copyright 2019 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.google.cloud.tasks.v2beta3;
+
+import static org.junit.Assert.fail;
+
+import com.google.api.gax.rpc.AlreadyExistsException;
+import com.google.common.base.Preconditions;
+import com.google.protobuf.ByteString;
+import java.io.IOException;
+import java.nio.charset.Charset;
+import java.util.logging.Level;
+import java.util.logging.Logger;
+import org.junit.BeforeClass;
+import org.junit.Test;
+
+public class CloudTasksSmokeTest {
+  private static final String PROJECT_ENV_NAME = "GOOGLE_CLOUD_PROJECT";
+  private static final String LEGACY_PROJECT_ENV_NAME = "GCLOUD_PROJECT";
+  private static final String LOCATION_ENV_NAME = "GOOGLE_CLOUD_LOCATION";
+  private static final String DEFAULT_LOCATION_ID = "us-central1";
+  private static final String QUEUE_ID = "default";
+  static Queue queue;
+
+  public static void main(String args[]) {
+    Logger.getLogger("").setLevel(Level.WARNING);
+    try {
+      createTask(getProjectId(), getLocationId(), QUEUE_ID);
+      System.out.println("OK");
+    } catch (IOException e) {
+      System.err.println("Failed with exception:");
+      e.printStackTrace(System.err);
+      System.exit(1);
+    }
+  }
+
+  @BeforeClass
+  public static void setupQueue() {
+    String queueName = QueueName.of(getProjectId(), getLocationId(), QUEUE_ID).toString();
+    try (CloudTasksClient client = CloudTasksClient.create()) {
+      Queue q = Queue.newBuilder().setName(queueName).build();
+      try {
+
+        queue = client.createQueue(LocationName.of(getProjectId(), getLocationId()), q);
+      } catch (AlreadyExistsException e) {
+        queue = client.getQueue(queueName);
+      }
+    } catch (IOException e) {
+      fail("Failed to create queue: " + e.getMessage());
+    }
+  }
+
+  @Test
+  public void run() {
+    main(null);
```

Comment:
minor - want to inline this?

Reply:
I followed the existing format for the generated SmokeTests ([example](https://github.com/googleapis/google-cloud-java/blob/master/google-cloud-clients/google-cloud-trace/src/test/java/com/google/cloud/trace/v2/TraceServiceSmokeTest.java))